app/views/duties/index.html.erb

```diff
@@ -60,9 +60,21 @@
                     <% place.each do |duty| %>
                       <% if duty[:name].nil? %>
                         <td class="schedule-name-empty" colspan="<%= duty[:colspan] %>">&nbsp;</td>
+                      <% elsif duty[:free] %>
+                        <td class="schedule-name-empty" colspan="<%= duty[:colspan] %>">
+                           <%= button_to "Grab !", {:controller => "duties", :action => "grab", :grab_duty => duty[:duty]}, {:method => :post, :remote => true} %>
```

Comment:
Use the new hash syntax. `{ :controller => 'duties', ... }` change to `{ controller: 'duties', ... }`